.github/scripts/generate_changelog.sh

```diff
@@ -11,7 +11,9 @@ mkdir -p changelog/revision
 cp openapi/v2/openapi-*.json changelog/revision/
 
 echo "Generating revision metadata file"
-revision_version=$(< openapi/v2/versions.json jq -r '.[]' | paste -sd ',' - | sed "s/,preview//")
+revision_version=$(< openapi/v2/versions.json jq -r '
+  .[] | select(. != "preview" and (endswith(".upcoming") | not))
+' | paste -sd ',' -)
```

Comment:
This change avoid to conside upcoming for the changelog for now. We will revert this change once we add support for upcoming to changelog.